src/include/planner/plannodes/insert_plan_node.h

```diff
@@ -79,6 +79,16 @@ class InsertPlanNode : public AbstractPlanNode {
       return *this;
     }
 
+
+    /**
+     * @param index_oid oid of insertion index that is to be added to this plan node
+     * @return builder object
+     */
+    Builder &AddIndexOid(catalog::index_oid_t index_oid) {
```

Comment:
Do we expect to add OIDs one at a time? It's going to come from the catalog, right? CatalogAccessor::GetIndexOids directly returns the vector for a given table.